### PR TITLE
Add details about weekly bug triage call in the README

### PR DESCRIPTION
**Pull request by humblec**
Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

### CODE REVIEW

**Comment by humblec** [comment]
@Madhu-1 @ajarr PTAL .

File: README.md [hunk 1]
```diff
@@ -91,6 +91,11 @@ if you are interested to contribute to this repo.
 
 Please submit an issue at: [Issues](https://github.com/ceph/ceph-csi/issues)
 
+## Weekly Bug Traige call
+
+We are conducting weekly bug triage call at our slack on every tuesday.
```
Comment (ajarr) [inline README.md]: s/tuesday/Tuesday/

"We conduct weekly bug triage calls at our slack channel on Tuesdays" ?

Comment (humblec) [reply README.md]: @ajarr Done.. thanks !

File: README.md [hunk 2]
```diff
@@ -91,6 +91,11 @@ if you are interested to contribute to this repo.
 
 Please submit an issue at: [Issues](https://github.com/ceph/ceph-csi/issues)
 
+## Weekly Bug Traige call
```
Comment (ajarr) [inline README.md]: Oops, missed this,
s/Traige/Triage/

Comment (humblec) [reply README.md]: @ajarr this is what happens when you attend a meeting and send PR at same time :).. Thanks

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (humblec) [reply README.md]: @Madhu-1 I deliberately didnt mention it. The reason being , the issue can be updated if there are some changes or can be used to provide status update..etc.

[the remark this answers is not in the document]
Comment (Madhu-1) [reply README.md]: okay, Thanks . are we planning to keep that issue open?

[the remark this answers is not in the document]
Comment (ajarr) [reply README.md]: > please mention the time slot and we can close the below-mentioned issue I believe

good point

[the remark this answers is not in the document]
Comment (ajarr) [reply README.md]: > @Madhu-1 I deliberately didnt mention it. The reason being , the issue can be updated if there are some changes or can be used to provide status update..etc.

It's much simpler if we give them a UTC time here. Keep changing it. I don't know if it's friendly for the contributor to click on the issue, scroll down through the comments and identify the timing

[the remark this answers is not in the document]
Comment (humblec) [reply README.md]: > okay, Thanks . are we planning to keep that issue open?

Yeah, lets keep it open for some time. That can be a metrics for the progress we make!

[the remark this answers is not in the document]
Comment (humblec) [reply README.md]: > > @Madhu-1 I deliberately didnt mention it. The reason being , the issue can be updated if there are some changes or can be used to provide status update..etc.
> 
> It's much simpler if we give them a UTC time here. Keep changing it PRs whenever needed. I think it'd be annoying for the contributor to click on the issue, scroll down through the comments and identify the right timing
>
Done

**Comment by humblec** [comment]
Merging this as we need more participation in the call at the earliest.

**Pull request merged**